### PR TITLE
Fixed fullscreen support, and added keyboard shortcuts.

### PR DESCRIPTION
**Pull request by IanMHoffman**
Allowed program to go full screen, making you able to use all of the programs menu items. 

Now supports keyboard shortcuts for:
Open Ctrl+O
New  Ctrl+N
Save Ctrl+S
Exit Ctrl+Q

### CODE REVIEW

File: PyPose.py [hunk 1]
```diff
@@ -72,17 +72,18 @@ def __init__(self):
...
         # build our menu bar  
         menubar = wx.MenuBar()
         prjmenu = wx.Menu()
-        prjmenu.Append(self.ID_NEW, "new") # dialog with name, # of servos
-        prjmenu.Append(self.ID_OPEN, "open") # open file dialog
-        prjmenu.Append(self.ID_SAVE,"save") # if name unknown, ask, otherwise save
-        prjmenu.Append(self.ID_SAVE_AS,"save as") # ask for name, save
+        prjmenu.Append(self.ID_NEW, "&New\tCtrl+N", "", wx.ITEM_NORMAL) # dialog with name, # of servos
+        prjmenu.Append(self.ID_OPEN, "&Open\tCtrl+O", "", wx.ITEM_NORMAL) # open file dialog
+	prjmenu.AppendSeparator()
```
Comment (mikeferguson) [inline PyPose.py:81]: Looks like you've got a tab here.

Comment (mikeferguson) [inline PyPose.py:81]: There are several more below as well.

**Comment by mikeferguson** [comment]
Looks good. If you fix up the tab issues we can merge it. Also, what OS are you using/testing on?

**Comment by IanMHoffman** [comment]
We are using Fedora 20.

**Comment by IanMHoffman** [comment]
Tabs are fixed, works great on my end.

**Pull request merged**